ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/ws/controller/KeyControllerV2.java

```diff
@@ -73,9 +73,11 @@ public KeyControllerV2(VerifierDataService verifierDataService) {
             @RequestParam CertFormat certFormat) {
         Instant now = Instant.now();
         List<ClientCert> dscs = verifierDataService.findDscs(since, certFormat, upTo);
+        HttpHeaders headers = CacheUtil.createExpiresHeader(
+                CacheUtil.roundToNextKeysBucketStart(now));
```

Comment:
`.headers(getKeysUpdatesHeaders(dscs, upTo, now))` already has everything in there. Why is this necessary? I think removing Cache-Control on line 78 should be enough.